pb_bss/evaluation/wrapper.py

```diff
@@ -267,6 +268,9 @@ def __init__(
                     8000: wide band (wb)
             enable_si_sdr: Since SI-SDR is only well defined for non-reverb
                 single-channel data, it is disabled by default.
+            compute_permutation: If True, assume the estimate is permuted
```

Comment:
change "is permuted" to "needs to be permuted"?

Reply:
Done